tests/fixture/mod.rs

```diff
@@ -4,13 +4,15 @@
 )]
 
 use orcapod::{
-    error::Result,
+    error::OrcaError,
     model::{Annotation, Pod, StreamInfo},
     store::{filestore::LocalFileStore, Store},
 };
-use std::{collections::BTreeMap, fs, ops::Deref, path::PathBuf};
+use std::{collections::BTreeMap, fs, ops::Deref, path::PathBuf, result};
 use tempfile::tempdir;
 
+pub type Result<T> = result::Result<T, OrcaError>;
```

Comment:
Given users might also want to use our alias for Result if they work a lot with OrcaError, let's treat it as publicly facing interface. Aside, for library usage we need to make it public anyway (please refer to the comment on error.rs)